ui/dashboards/src/components/DashboardToolbar.tsx

```diff
@@ -40,14 +62,20 @@ export const DashboardToolbar = () => {
               <Button variant="contained">Save</Button>
             </Stack>
           </Box>
-          <Button
+          <Box
             sx={{
               alignSelf: 'flex-end',
             }}
           >
-            <AddIcon sx={{ marginRight: '8px' }} />
-            Add Panel
-          </Button>
+            <Button onClick={onAddGroup}>
+              <AddIcon sx={{ marginRight: '8px' }} />
+              Add Group
+            </Button>
+            <Button onClick={onAddPanel}>
+              <AddIcon sx={{ marginRight: '8px' }} />
+              Add Panel
+            </Button>
```

Comment:
I think you want to use the `startIcon` property on `Button` here rather than putting it into children. Should take care of the margin for you without needing custom styles.